lwallet/Command.hs

```diff
@@ -111,6 +113,20 @@ switch = lexeme $ positive $> True <|>
     positive = text "+" <|> text "y" <|> text "yes"
     negative = text "-" <|> text "n" <|> text "no"
 
+stakeholderId :: Parser StakeholderId
+stakeholderId = lexeme $ do
+    str <- many1 alphaNum
+    case decodeTextAddress (toText str) of
```

Comment:
@naryl you may have a problem, because some code uses `Parsec Text`, but other code uses `Parsec String`.
I am currently getting rid of `Parsec String`.
Unfortunately, I was in hurry and didn't do it in a separate commit :(
If this PR can wait, it's better to wait until #1342 is ready and merged.